2019/python/lar_constraints.py

```diff
@@ -983,6 +991,33 @@ def v696_const(self, row):
 			row["aus_4"] = ""
 			row["aus_5"] = ""
 
+		#number of reported systems must match the number of reported results
```

Comment:
What wasn't working about the short loop from the old code? It looks like adding the new codes should make that work.
```
#number of reported systems must match the number of reported results
		result_enums = ("1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14","15", "16")
		for i in range(1, len(aus_sys)):
			if aus_sys[i] in ("1", "2", "3", "4", "5") and aus_results[i] not in result_enums:
				row["aus_result_"+str(i+1)] = random.choice(result_enums[:-1])
```